packages/components/src/Enumeration/Header/Header.component.js

```diff
@@ -34,6 +34,7 @@ function getAction(action, index) {
 	}
 	return (
 		<Action
+			{...action}
 			key={`${index}-enum-header-action`}
 			label={action.label}
 			icon={action.icon}
```

Comment:
You can now remove all the props that you pass through as they are already spread